cw_monero/lib/monero_subaddress_list.dart

```diff
@@ -143,8 +147,7 @@ abstract class MoneroSubaddressListBase with Store {
                     label.toLowerCase() == 'Primary account'.toLowerCase()
                 ? 'Primary address'
                 : label);
-      })
-        .toList();
+      }).toList().reversed.toList();
```

Comment:
Why reverse it here?

Reply:
otherwise addresses appear upside down in the receive screen